src/Makefile

```diff
@@ -68,7 +68,7 @@ endif
 
 ifneq (,$(findstring pgi,$(COMPILER)))
 	ifeq ($(PERFORMING_CONDA_BUILD),True)
-		CPPFLAGS += -tp=px
+		CPPFLAGS += -tp=piledriver
```

Comment:
Even though this is AMD the instruction set used would still be applicable to Intel?

Reply:
Of course... they copy from each other :)
Of the available options, piledriver is the first one to have AVX and FMA support.